app/controllers/idv/doc_auth_controller.rb

```diff
@@ -54,5 +57,13 @@ def do_meta_refresh(meta_refresh_count)
     def flow_session
       user_session['idv/doc_auth']
     end
+
+    def check_for_outage
+      vendor_status = VendorStatus.new
+      if vendor_status.any_ial2_vendor_outage?
+        session[:vendor_outage_redirect] = "IdV: #{current_step}"
```

Comment:
The vendor outage controller doesn't know if the outage is idv or not, so would have to check for multiple session keys. Is that better? We may also want to know if a user was in the flow when we flipped the config and this will track the step they were on, though I recognize that will be uncommon.

Or are you suggesting more like this?:
```
        session[:vendor_outage_redirect] = current_step
        session[:redirect_from_idv] = true
```

Reply:
Yes this is what I was imagining:

```
        session[:vendor_outage_redirect] = current_step
        session[:redirect_from_idv] = true
```

Reply:
OK, done in c60cafe. Let me know if not what you were thinking.

Reply:
Yup! That's what I meant! Thank you